### PR TITLE
Homework №2. Object gauss.

### PR DESCRIPTION
**Pull request by kkulahin**
### _Homework №2_

```javascript
function randn_bm() {
    let u = 0, v = 0;
    while (u === 0) u = Math.random();
    while (v === 0) v = Math.random();
    return Math.sqrt(-2.0 * Math.log(u)) * Math.cos(2.0 * Math.PI * v) * 2 | 0;
}
```

- Using provided function generate 10 numbers. Create object with the keys related to the generated numbers and value as a quantity of corresponding number. Display numbers distribution as a table using template literal string.

- Example of object: {0: 5, 1: 3, 2: 2}

image

### CODE REVIEW

File: hw_2/script.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    return resultObject;
+}
+
+function drawTable(obj) {
+    console.log(obj);
+    console.log("+---------+---------+");
+    console.log("|  Number | Count   |");
+    console.log("+---------+---------+");
+    Object.getOwnPropertyNames(obj)
+        .sort((x, y) => x - y)
```
Comment (MishkanJigan) [inline hw_2/script.js]: Do we need this sort function in our task?

File: hw_2/script.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            resultObject[elem] += 1;
+        } else {
+            resultObject[elem] = 1;
+        }
+    }
+    return resultObject;
+}
+
+function drawTable(obj) {
+    console.log(obj);
```
Comment (MishkanJigan) [inline hw_2/script.js]: Do we need `console.log(obj)`?

File: hw_2/script.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    while (u === 0) u = Math.random();
+    while (v === 0) v = Math.random();
+    return (Math.sqrt(-2.0 * Math.log(u)) * Math.cos(2.0 * Math.PI * v) * 2) | 0;
+}
+
+function fillAndGroupRandom(count, randomFunc) {
+    const resultObject = {};
+    let wCount = count + 1;
+
+    while ((wCount -= 1)) {
```
Comment (MishkanJigan) [inline hw_2/script.js]: What is the wCount? Сan we think about a better name for the variable?

File: hw_2/script.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
+function randn_bm() {
+    let u = 0,
+        v = 0;
+    while (u === 0) u = Math.random();
+    while (v === 0) v = Math.random();
+    return (Math.sqrt(-2.0 * Math.log(u)) * Math.cos(2.0 * Math.PI * v) * 2) | 0;
+}
+
+function fillAndGroupRandom(count, randomFunc) {
```
Comment (MishkanJigan) [inline hw_2/script.js]: This function return to us a new object which you generate from rundom function, the function name is a bit opaque.

File: hw_2/script.js (hunk 3 above)
Comment (kkulahin) [reply hw_2/script.js]: https://github.com/BlackTruth/javascript/blob/0f090223bfbe9aef482c5d350255c17995e68193/hw_2/script.js#L11-L12
Removed wCount variable.

File: hw_2/script.js (hunk 1 above)
Comment (kkulahin) [reply hw_2/script.js]: https://github.com/BlackTruth/javascript/blob/0f090223bfbe9aef482c5d350255c17995e68193/hw_2/script.js#L27-L28
Removed sorting object.

File: hw_2/script.js (hunk 2 above)
Comment (kkulahin) [reply hw_2/script.js]: https://github.com/BlackTruth/javascript/blob/0f090223bfbe9aef482c5d350255c17995e68193/hw_2/script.js#L23-L24
Removed console.log(obj).

File: hw_2/script.js (hunk 4 above)
Comment (kkulahin) [reply hw_2/script.js]: https://github.com/BlackTruth/javascript/blob/0f090223bfbe9aef482c5d350255c17995e68193/hw_2/script.js#L9
Renamed function.

**Pull request merged**